src/app/example_configs/config.yaml

```diff
@@ -175,6 +175,7 @@ plugin:
       username: plugin_admin
     directory: ./plugins
     host_env_vars: []
+    capture_streams: true
```

Comment:
In the example, let's make this `false`. That way we don't have extra files being generated in our development environment.

Reply:
done